docs/sourcecontrol/overview.md

```diff
@@ -115,6 +115,38 @@ If you open a folder that is a Git repository and begin making changes, VS Code
 
 Merge conflicts are recognized by VS Code. Differences are highlighted and there are inline actions to accept either one or both changes. Once the conflicts are resolved, stage the conflicting file so you can commit those changes.
 
+## 3 Way Merge Editor
```

Comment:
Can we add short paragraph that explains what a merge editor is and why should people use it?

Reply:
Have a look at https://github.com/microsoft/vscode-docs/pull/6131, which added an opening sentence to tie the above merge conflicts section to the 3-way merge editor

Reply:
Yes, I really like that!
I would not add more details yet. Let's wait and see how do the users react :)
Thanks all!